src/pip/_internal/commands/freeze.py

```diff
@@ -61,7 +72,7 @@ def add_options(self) -> None:
             action="store_true",
             help=(
                 "Do not skip these packages in the output:"
-                " {}".format(", ".join(DEV_PKGS))
+                " {}".format(", ".join(_dev_pkgs()))
```

Comment:
Just a minor note, it’d be a good idea to add some sorting here so the order package appear is more stable. But honestly nobody care that much about these help messages anyway so it’s fine to keep things as-is.